### ISSUE TITLE
act_critic.py中actor_critic类中，updata和updata2为何一个to device一个没有

### ISSUE THREAD

**Issue by Wesley273** [pengguo318/MaDRLAM#4]
`
    def updata(self,task_action_pro,reward1,q,lr):

        opt = optim.Adam(self.actor1.parameters(), lr)

        pro = torch.log(task_action_pro)

        loss = torch.sum(pro, dim=1)

        score = reward1 - q

        score = score.detach()

        loss = score * loss

        loss = torch.sum(loss) / configs.batch

        opt.zero_grad()
        loss.backward()
        opt.step()

    def updata2(self, p_action_pro, reward1, q,lr):

        opt = optim.Adam(self.actor2.parameters(), lr)

        pro = torch.log(p_action_pro)

        loss = torch.sum(pro, dim=1).to(DEVICE)

        score = reward1 - q

        score = score.detach().to(DEVICE)

        loss = score * loss

        loss = torch.sum(loss) / configs.batch

        opt.zero_grad()
        loss.backward()
        opt.step()
`
